test/lib/oauth.coffee

```diff
@@ -3,27 +3,23 @@ bond = require "bondjs"
 request = require "request"
 OAuth = require "../../lib/oauth"
 config = require "../config"
-helper = require "../helper"
-helper.stubOAuth(3)
+fixture = require "../fixtures"
 
 describe "OAuth", ->
   describe "getToken", ->
     before -> @spy = bond(request, "post").through()
+    before -> fixture.load "oauth"
     before (done) ->
       oauth = new OAuth config
       oauth.getToken (@err, @token, @secret) =>
         done @err
     after -> @spy.restore()
 
-    it "should return an OAuth token", ->
+    it "should make an Authorized request to return an OAuth token without error", ->
       assert.equal @err, null
       assert @token
       assert @secret
 
-    it "should have an Authorization header with the consumer key", ->
-      regex = new RegExp "#{config.consumerKey}"
-      assert regex.test @spy.calledArgs[0][0].headers.Authorization
```

Comment:
This header gets asserted by the fixture loaded by nock so a successful request means the header matched